README.rst

```diff
@@ -6,11 +6,11 @@
     :target: https://pypi.python.org/pypi/django-comments-dab/
     :alt: django
 
-.. image:: https://codecov.io/gh/radi85/Comment/branch/main/graph/badge.svg?token=JBorE9i0De
+.. image:: https://codecov.io/gh/radi85/Comment/branch/master/graph/badge.svg?token=JBorE9i0De
```

Comment:
Shouldn't this be for the `develop` branch?

```suggestion
.. image:: https://codecov.io/gh/radi85/Comment/branch/develop/graph/badge.svg?token=JBorE9i0De
```

Reply:
I think this should be the stable version which is master, right?

Reply:
i think `readthedocs` give you the option to go to enable different versions and set `stable` as the default version. keeping this to `develop` would help us see the documentation builds instantly similar to test badges.